doc/classes/Node.xml

```diff
@@ -994,6 +994,7 @@
 		</constant>
 		<constant name="DUPLICATE_USE_INSTANCING" value="8" enum="DuplicateFlags">
 			Duplicate using instancing.
+			(An instance stays linked to the original so when the original changes, 			the instance changes too.)
```

Comment:
There's a lot of whitespace between the command and "the" which should be removed.
The comment is good, but I think the parentheses are not necessary.

Reply:
Yeah sorry I meant to reference godotengine/godot-docs#2770.
Should I create a new pull request with removed whitespace and parenthesis?
Sorry if that question sounded silly I'm kinda new to open source.

Reply:
No need to make a new pull request, you can modify this one following this workflow: http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request

Reply:
done